static/install/compose

```diff
@@ -156,11 +158,10 @@ services:
       - _APP_MIGRATIONS_FIREBASE_CLIENT_ID
       - _APP_MIGRATIONS_FIREBASE_CLIENT_SECRET
       - _APP_ASSISTANT_OPENAI_API_KEY
-
   appwrite-console:
     <<: *x-logging
     container_name: appwrite-console
-    image: appwrite/console:5.2.27
+    image: appwrite/console:5.2.58
```

Comment:
Is this the latest version 🤔  We use `5.3.8` on cloud

Reply:
Cloud is ahead of 1.6.2 and there were issues with 5.3.x when this release was being prepared.